memor/response.py

```diff
@@ -249,6 +250,28 @@ def from_json(self, json_object: Union[str, Dict[str, Any]]) -> None:
             date_modified = datetime.datetime.strptime(loaded_obj["date_modified"], DATE_TIME_FORMAT)
         except Exception:
             raise MemorValidationError(INVALID_RESPONSE_STRUCTURE_MESSAGE)
+        _validate_string(message, "message")
+        if score:
+            _validate_probability(score, "score")
+        if temperature:
+            _validate_pos_float(temperature, "temperature")
+        if tokens:
+            _validate_pos_int(tokens, "tokens")
+        if inference_time:
+            _validate_pos_float(inference_time, "inference_time")
+        _validate_string(model, "model")
+        _validate_message_id(_id)
+        _validate_string(memor_version, "memor_version")
+        return message, score, temperature, tokens, inference_time, model, role, memor_version, _id, date_created, date_modified
```

Comment:
The `Response` class has many parameters, so returning them in this format is not a good approach. I suggest returning a dictionary instead.

Reply:
fixed in [9523075](https://github.com/openscilab/memor/pull/120/commits/9523075ef2ebd4a782024e43d9cd231834bd51a5)